HOWTO-make-a-release.md

```diff
@@ -0,0 +1,377 @@
+subsequent release.  Alternatively if you want to perform one release at a
+time or copy/move the files manually, see below.
+
+The do-release.pl script will display the commands you will need to issue to
+send the announcement emails later.  Keep a note of those commands for
+future reference.
+
+Verify that the tarballs are available via FTP:
+
+        ftp://ftp.openssl.org/source/
+
+And that they are ready for the website:
+
+        ls /var/www/openssl/source
+
+*For OpenSSL 3.0 and on*, push your local changes to the public source repo
+as instructed by `dev/release.sh`.  You may want to sanity check the pushes
+by inserting the `-n` (dry-run) option.
+
+*For OpenSSL before 3.0*, simply push your local changes to the public
+source repo, and please do remember to push the release tags as well, which
+is done separately with the `--tags` option.  You may want to sanity check
+the pushes by inserting the `-n` (dry-run) option.
+
+## Updating the website
+
+Push the website changes you made earlier to the OpenSSL website repo.  When
+you do this, the website will get updated and a script to flush the Akamai CDN
+cache will be run.  You can look at things on www-origin.openssl.org; the
+CDN-hosted www.openssl.org should only be a few minutes delayed.
+
+# Post-publishing tasks
+
+## Check the website
+
+Verify that the release notes, which are built from the CHANGES file in the
+release, have been updated.  This is done automatically by the commit-hook,
+but if you see a problem, try the following steps on `dev.openssl.org`:
+
+    cd /var/www/openssl
+    sudo -u openssl -H make relupd
+    sudo -u openssl ./bin/purge-one-hour
+
+Wait for a while for the Akamai flush to work (normally within a few minutes).
+Have a look at the website and news announcement at:
+
+-   <https://www.openssl.org/>
+-   <https://www.openssl.org/news/>
+
+Check the download page has updated properly:
+
+-   <https://www.openssl.org/source/>
+
+Check the notes look sensible at:
+
+-   <https://www.openssl.org/news/newslog.html>
+
+Also check the notes here:
+
+-   <https://www.openssl.org/news/openssl-1.0.2-notes.html>
+-   <https://www.openssl.org/news/openssl-1.1.0-notes.html>
+-   <https://www.openssl.org/news/openssl-1.1.1-notes.html>
+
+## Send the announcement mail
+
+Send out the announcements.  Generic release announcement messages will be
+created automatically by the build script and the commands you need to use
+to send them were displayed when you executed do-release.pl above.
+These should normally be sent from the openssl account.  These are sent to
+openssl-users, openssl-project, and openssl-announce.
+
+If do-release.pl was used with `--move` be sure to move the announcement
+text files away from the staging directory after they have been sent.  This
+is done as follows (with VERSION replaced with the version of OpenSSL to
+announce):
+
+        sudo -u openssl \
+            mv ~openssl/dist/new/openssl-VERSION.txt.asc ~openssl/dist/old
```

Comment:
It's a bit odd because the do-release command tells you what to do next and gives 2 commands for you to execute (one to to send the emails and one to mv the announcement file) - this text repeats it here but only gives one of them. I think it would be helpful to give the other one as well in case you forgot to take note when the script was run earlier.

Reply:
It sounds like `do-release.pl` could do with a few fixes, such as instructing *at the end*

Reply:
I've added the suggested command